### PR TITLE
Release notes 0.13.0

### PR DESCRIPTION
**Pull request by fedepaol**
Add the release notes for the upcoming version.

### CODE REVIEW

File: website/content/release-notes/_index.md [hunk 1]
```diff
@@ -3,6 +3,51 @@ title: Release Notes
 weight: 8
 ---
 
+## Version 0.13.0
+
+New Features:
+
+- CRD support! A long awaited feature, MetalLB is now configurable via CRs. As a side effect of this new configuration,
```
Comment (msherif1234) [inline website/content/release-notes/_index.md:10]: link to sample configs will be great to ease migration from configmap to CR, also I assume major update to [doc](https://metallb.universe.tf/configuration) to reflect he new config

File: website/content/release-notes/_index.md [hunk 2]
```diff
@@ -25,7 +70,7 @@ New Features:
   ([PR #1065](https://github.com/metallb/metallb/pull/1065))
 
 - In FRR mode, it is possible to have a BGP session paired with a BFD session
-  for quicker path failure detection.
+  for <https://github.com/metallb/metallb/pull/1230://github.com/metallb/metallb/pull/1230quicker> path failure detection.
```
Comment (liornoy) [inline website/content/release-notes/_index.md:73]: nit: accidental paste?

Comment (fedepaol) [reply website/content/release-notes/_index.md:73]: yes :-)

File: website/content/release-notes/_index.md [hunk 3]
```diff
@@ -3,6 +3,58 @@ title: Release Notes
...
 
+## Version 0.13.0
+
+New Features:
+
+- CRD support! A long awaited feature, MetalLB is now configurable via CRs. As a side effect of this new configuration,
+  it's now possible to choose to advertise addresses in L2 mode, BGP mode, both or just allocate the IP without advertising it.
+  On top of that, validating webhooks will ensure the validity of the configuration upfront, without needing to check the logs.
+  ([PR #1237](https://github.com/metallb/metallb/pull/1237), [PR #1245](https://github.com/metallb/metallb/pull/1245))
+  Please check the docs on how to use the converter tool to generate the CRs from your configmap.
```
[the remark this answers is not in the document]
Comment (cgoncalves) [reply website/content/release-notes/_index.md:14]: OK, I read now the item under the "Changes in behavior" section down below. Perhaps consider instead pointing readers to that item.

File: website/content/release-notes/_index.md [hunk 4]
```diff
@@ -3,6 +3,58 @@ title: Release Notes
...
+
+- the biggest change is the introduction of CRDs and removing support for the configuration via ConfigMap. In order to ease the transition
+  to the new configuration, we provide a conversion tool from ConfigMap to resources (TODO Add link here)
+
+- the internal architecture was radically changed in order to accomodate CRDs, so please do not hesitate to [file an issue](https://github.com/metallb/metallb/issues).
+
+- The `AvoidBuggyIPs` flag was removed in order to reduce the api surface a bit. The same result can be achieved using ranges of IPs instead of
+  the CIDR annotation.
+
+- The metallb images from dockerhub are **deprecated**. From this release, only the images on quay.io will be supported and updated.
```
Comment (cgoncalves) [inline website/content/release-notes/_index.md:38]: A direct reference to the MetalLB organization listing all the container images would be best. Consider adding a link to https://quay.io/organization/metallb

File: website/content/release-notes/_index.md [hunk 5]
```diff
@@ -3,6 +3,58 @@ title: Release Notes
 weight: 8
 ---
 
+## Version 0.13.0
+
+New Features:
+
+- CRD support! A long awaited feature, MetalLB is now configurable via CRs. As a side effect of this new configuration,
+  it's now possible to choose to advertise addresses in L2 mode, BGP mode, both or just allocate the IP without advertising it.
```
Comment (cgoncalves) [inline website/content/release-notes/_index.md:11]: Consider splitting the side effect part into another item, keeping the converter tool in this first item.

**Review by cgoncalves — CHANGES_REQUESTED** [review]

File: website/content/release-notes/_index.md (hunk 4 above)
Comment (fedepaol) [reply website/content/release-notes/_index.md]: done

**Comment by cgoncalves** [comment]
LGTM

**Pull request merged**